op-challenger/game/fault/contracts/outputbisectiongame.go

```diff
@@ -24,7 +24,7 @@ type OutputBisectionGameContract struct {
 }
 
 func NewOutputBisectionGameContract(addr common.Address, caller *batching.MultiCaller) (*OutputBisectionGameContract, error) {
-	contractAbi, err := bindings.OutputBisectionGameMetaData.GetAbi()
+	contractAbi, err := bindings.FaultDisputeGameMetaData.GetAbi()
```

Comment:
As a follow up we probably should rename `OutputBisectionGameContract` to `FaultDisputeGameContract` to match the actual contract name.